Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "streaming-stats"
-version = "0.2.2"  #:version
+version = "0.2.3"  #:version
```

Comment:
Please remove this version bump. I do this myself as part of the release process.